test/com/google/enterprise/adaptor/database/UniqueKeyTest.java

```diff
@@ -104,6 +104,32 @@ public void testNameRepeatsNotAllowed() {
   }
 
   @Test
+  public void testCaseSensitiveNameRepeatsAllowed() {
+    UniqueKey uk = new UniqueKey("num:int,Num:string");
+    assertEquals(2, uk.numElementsForTest());
+    assertEquals("num", uk.nameForTest(0));
+    assertEquals("Num", uk.nameForTest(1));
+    assertEquals(UniqueKey.ColumnType.INT, uk.typeForTest(0));
```

Comment:
As a separate CL, what would you think about using H2 with prepared statements to test things like this through the public interface, rather than having all these test-specific methods in UniqueKey?